sql/src/test/java/io/crate/integrationtests/PartitionedTableIntegrationTest.java

```diff
@@ -2142,4 +2142,18 @@ public void testRefreshIgnoresClosedPartitions() {
             is("1| 1\n")
         );
     }
+
+    @Test
+    public void testWhereOnPartitionedAndOtherColumn() {
+        execute("create table t (p string primary key, v string) " +
+                "partitioned by (p) " +
+                "with (number_of_replicas = 0)");
+        ensureYellow();
```

Comment:
ensureYellow isn't needed. No shards are created at this point

Reply:
yep sorry, copy-pasted :-(